packages/opentelemetry-core/src/trace/TracerDelegate.ts

```diff
@@ -55,6 +55,15 @@ export class TracerDelegate implements types.Tracer {
     );
   }
 
+  bind<T>(target: T, span?: types.Span): T {
+    return this._currentTracer.bind.apply(
+      this._currentTracer,
+      // tslint:disable-next-line:no-any
+      arguments as any
+      // tslint:disable-next-line:no-any
+    ) as any;
```

Comment:
Is `any` needed here? Can you do `as unknown as T`?

Reply:
True, i changed it